zenodo/modules/deposit/static/json/zenodo_deposit/deposit_form.json

```
@@ -591,9 +603,17 @@
                     "value": "10.13039/501100001602",
                     "name": "Science Foundation Ireland (IE)"
                   },
+                  {
+                    "value": "10.13039/100001345",
+                    "name": "Social Science Research Council (EN)"
+                  },
                   {
                     "value": "10.13039/100004440",
                     "name": "Wellcome Trust (UK)"
+                  },
+                  {
+                    "value": "10.13039/501100000024",
+                    "name": "T\\u00fcrkiye Bilimsel ve Teknolojik Ara\\u015ftirma Kurumu (TR)"
```

```suggestion
                    "name": "T\u00fcrkiye Bilimsel ve Teknolojik Ara\u015ftirma Kurumu (TR)"
```

I see in another unicode string above that it has a single backslash. Can you give it a try locally and check that they're displayed in the form?